dart/tool/java/src_gen_java.dart

```diff
@@ -3,6 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 /// A library to generate Java source code.
+/// See [JavaGenerator].
```

Comment:
Markdown will emit these as one line. I would either add a blank line between the two lines if that is your intention, or just write this comment as one line.